morpheushelper/cogs/mod.py

```diff
@@ -406,11 +406,12 @@ async def unmute(self, ctx: Context, user: Union[Member, User, int], *, reason:
     @commands.command()
     @Permission.kick.check
     @guild_only()
-    async def kick(self, ctx: Context, member: Member, *, reason: str):
+    async def kick(self, ctx: Context, member: Union[Member, int], *, reason: str):
         """
         kick a member
         """
-
+        member_on_server: bool = isinstance(member, Member)
+        member: Member = await self.get_user(ctx.guild, member)
```

Comment:
Maybe another var name like user makes more sense because it doesn't overload the method param.
Also the type is a `Member` or a `User` so the typing should be like this:
```python
user: Union[Member, User] = await self.get_user(ctx.guild, member)
```